debian/cloudstack-agent.install

```diff
@@ -18,12 +18,11 @@
 /etc/cloudstack/agent/agent.properties
 /etc/cloudstack/agent/environment.properties
 /etc/cloudstack/agent/log4j-cloud.xml
+/etc/default/cloudstack-agent
 /etc/profile.d/cloudstack-agent-profile.sh
 /etc/logrotate.d/cloudstack-agent
-/etc/init.d/cloudstack-agent
 /usr/bin/cloudstack-setup-agent
```

Comment:
Yes, that is true. I 'stole' this from other packages which still do the same.

In debian/rules I would have to make the decision to install these or not, but I wasn't really sure how to do so.